src/Blubrry/REST/Resource/Social.php

```diff
@@ -49,6 +68,85 @@ public function updateListing($program_keyword, $body) {
     public function getSocial($program_keyword, $params) {
         $path = '/2/social/' . $program_keyword . '/get-social-options.json';
 
+        $social_options = array();
+        $social_options['social-options'] = array();
+
+        $social_types=array('twitter', 'youtube', 'facebook');
+        $required_fields=array('social-id', 'social-image', 'social-title');
+        $attributes=array(
+            'html' => array(
+                'required' => array(),
+                'optional' => array('content'),
+            ),
+            'Input-text' => array(
```

Comment:
@lucashillebrandt Is that correct? Only this specific case needs to have the I in uppercase?

Reply:
Nope, changed to lowercase. Resolved.